cernopendata/modules/fixtures/data/docs/delphi-news-first-release/delphi-news-first-release.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "author": "DELPHI Collaboration",
+    "body": {
+      "content": "delphi-news-first-release.md",
+      "format": "md"
+    },
+    "date_published": "2024-06-26",
```

Comment:
Since we don't have a date, and since it may take some months before everything is ready, perhaps we could take out this news for now and add it at the very end?  (This is what we are usually doing when preparing large releases, the news announcement item is added only later when everything is ready.)